scala-steward.conf

```diff
@@ -1,6 +1,6 @@
 pullRequests.grouping = [
-  { name = "gu", "title" = "chore(deps): Guardian library updates", "filter" = [{"group" = "com.gu"}] },
-  { name = "aws", "title" = "chore(deps): AWS dependency updates", "filter" = [{"group" = "software.amazon.awssdk"}, {"group" = "com.amazonaws"}] },
+  { name = "gu", "title" = "chore(deps): Guardian library updates", "filter" = [{"group" = "com.gu"}], {"group" = "com.gu.*"}] },
+  { name = "aws", "title" = "chore(deps): AWS dependency updates", "filter" = [{"group" = "software.amazon.*"}, {"group" = "com.amazonaws"}, {"group" = "com.amazonaws.*"}] },
```

Comment:
@vlbee pointed out to me that artifacts like `"software.amazon.kinesis" % "amazon-kinesis-client"` were being included in _"**Non**-AWS dependency updates"_ (like https://github.com/guardian/mobile-notifications-content/pull/150) - and that's incorrect, we would expect that to go into the AWS updates.